cli/cmd/policy.go

```diff
@@ -286,6 +287,21 @@ func policyTable(policies []api.Policy) (out [][]string) {
 	sevThreshold, _ := severityToProperTypes(policyCmdState.Severity)
 
 	for _, policy := range policies {
+		// normalize policy.Enabled
+		var enabled string
+		if policy.Enabled == nil { // nil ptr check
+			enabled = "unknown"
```

Comment:
See this comment here regarding the Policy struct:
```
// Use of pointers intentional here for optimal json.Marshal() behavior
// This permits distinction between bool/int zero values vs. missing/unspecified
type Policy struct {
	ID           string                 `json:"policy_id,omitempty" yaml:"policy_id"`
	Title        string                 `json:"title,omitempty" yaml:"title"`
	Enabled      *bool                  `json:"enabled,omitempty" yaml:"enabled"`
	AlertEnabled *bool                  `json:"alert_enabled,omitempty" yaml:"alert_enabled"`
	Frequency    string                 `json:"eval_frequency,omitempty" yaml:"eval_frequency"`
	Severity     string                 `json:"severity,omitempty" yaml:"severity"`
	QueryID      string                 `json:"lql_id,omitempty" yaml:"lql_id"`
	AlertProfile string                 `json:"alert_profile,omitempty" yaml:"alert_profile"`
	Limit        *int                   `json:"limit,omitempty" yaml:"limit"`
	Description  string                 `json:"description,omitempty" yaml:"description"`
	Remediation  string                 `json:"remediation,omitempty" yaml:"remediation"`
	Properties   map[string]interface{} `json:"properties,omitempty" yaml:"properties"`
}
```

For booleans and integers it's important to use pointers such that `Marshal` can differentiate between something that's unspecified vs. it's Zero value.  This is critical for `omitempty` behavior.  

This post here does a good job of explaining the behavior:
https://www.sohamkamani.com/golang/omitempty/

Reply:
The only reason why you are adding 1) `omitempty` and 2) making the primitives pointers is because you want to re-use this struct for POST and PATCH requests but Create/POST requests do have required fields to be present and yes, Update/PATCH don't. I think we should be explicit and separate both structs, if not, the consumer will have to always validate that the pointers have content before accessing it, otherwise the app will crash.
```go
if myPolicy.Enabled != nil {
    // use Enabled field
    if myPolicy.Enabled {
        // policy is enabled
    }
}
```
Instead of doing that ☝🏽 , we could define the PATCH payload as a separate/hidden struct that the API package will use internally.

Reply:
Yes, this is a general purpose Policy struct use for GET / POST / PATCH.  If we prefer to separate out structures and use some internally then this is a possibility...